controllers/argocd/keycloak.go

```diff
@@ -1119,6 +1191,30 @@ func deleteKeycloakConfigForOpenShift(cr *argoprojv1a1.ArgoCD) error {
 		return err
 	}
 
+	err = deleteOAuthClient(cr)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// Delete OpenShift OAuthClient
+func deleteOAuthClient(cr *argoprojv1a1.ArgoCD) error {
```

Comment:
We can pass in `cfg` as an argument to avoid creating it in this funciton.

Reply:
Ack, Thanks :)